Database/Updates/World/04-06-20-2017-alterPostionTable-stored.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `ace_world`.`ace_position`
+  DROP COLUMN `landblock` ,
+  ADD COLUMN `landblock` INT(5) UNSIGNED AS ( ((landblockraw >> 16) & 0xFFFF)) STORED,
```

Comment:
I guess you will just not respect my wishing that Mogwai review my work.   Your treatment of me is unacceptable to me and I do not wish you to review my work.   I have made my wishes very clear and I have asked you nicely on at least 10 occasions.   "Please allow Mogwai to review my work".   As to your point, for loading, it would be very easy for you to drop the column to load.   On query, virtual query on that key to load 1400 records, 4 seconds.   Stored sub 1 second.   I am not changing this as it impacts the function that supports it.   I don't want to spend my free time with someone constantly who is critical of everything I do.   If you have to review my work - then my time on this project is at an end.   I will just work on a copy in my fork.   I will not submit anything to this project if you have to review it.   I have been working for almost 40 years and I have never been treated with such disrespect, or worked with anyone that would not discuss topics.   Your default position is that you make the final call end of story.   I am not going to do that.